### ISSUE TITLE
More complete example for web-based use

### ISSUE THREAD

**Issue by fiammybe** [dflydev/dflydev-embedded-composer#23]
Hi, 
I'm trying to use composer as part of a web page in the administration section of ImpressCMS (https://github.com/impresscms/impresscms). 
We are rewriting key parts of ImpressCMS , a PHP-based CMS that is aimed primarily at people that do not use command-lines and which runs well on shared hosting. Instead of reinventing the wheel, we would like to use composer as package manager for our ImpressCMS addon packages. Part of that is already working from the command-line, but now I wanted to get that available from a web page in the administration section as well.

Basically, I want to be able to do the following things with Composer :
- install a package from packagist
- uninstall a package
- update a package
- list installed packages and their versions
- check for updates from the installed packages

I saw there were 'install, update and dumpautoload' commands defined, but I can't seem to get a minimal example up and running that gets me a working composer with a standard composer.json in the root, and work from there. Could you point me out where I can find such an example (our where it is used in that way)?
